build/RELEASING.md

```diff
@@ -1,9 +1,34 @@
 # Release process
 
-Only for Maintainers.
+**Only for Maintainers.**
 
- 1. Decide the tag name (version name) to be released. e.g. 1.4.0-beta.1,
-    1.0.0-rc9.7 etc.
+ 1. Decide the component(s) and tag name (version name) to be released.
+
+    Notes:
+
+       * There are different categories of packages. Check the project file for
+         what you want to release and look for `MinVerTagPrefix`.
+
+         * `core-`: Core packages. These packages are defined\goverened by the
+           OpenTelemetry Specification and have released stable versions. They
```

Comment:
Is this statement true? For example https://github.com/open-telemetry/opentelemetry-dotnet/blob/1fb00d9a8a9c810fbe4e36ee8bb52a3f965e73f6/src/OpenTelemetry.Api.ProviderBuilderExtensions/OpenTelemetry.Api.ProviderBuilderExtensions.csproj#L7 says it is a core component, it is not defined/governed by the spec.

Reply:
I pushed updates to try and make everything clear (and to address the below two comments). Please check it out and LMK what you think.

Reply:
LGTM, thanks!